calcom/cal.com
# fix: `videoCallUrl` not updating when rescheduling with a broken Calendar integration

## What does this PR do?
Fixes, `videoCallUrl` not updating when rescheduling with a broken Calendar integration

- Added an asssertion that verifies that it works now.
- Added assertion to verify appsStatus also in email
- Started using node-html-parser to easily test more things in email HTML

More details https://threads.com/34542266165?s=iejN9Cku3XDgLFAaAXrKBo


## Type of change


## How should this be tested?
- Connect Google Calendar and setup Daily Video
- Book a meeting with an event-type that uses the Google Calendar and has Daily Video as location
- Go to Credential table and modify refresh_token and access_token for Google Calendar so that it can no longer fix itself
- Reschedule the previous booking and notice that it is giving the previous Cal Video URL instead of a new one corresponding to the new booking UID.

## Mandatory Tasks
